index.html

```diff
@@ -13,20 +13,20 @@
 </head>
 <body>
     <header class="app-bar container-h">
-        <h2>My Logo</h2>
+        <a href="#" class="logo">My Logo</a>
         <button class="simple-button"><img src="images/Union.svg" alt="Burger menu"></button>
     </header>
-    <main class="app-headline container-v">
+    <section class="app-headline container-v">
         <h1>I’m Mariane Glad to see you!</h1>
         <p class="simple-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
         <span class="social-media">LET’S CONNECT</span>
-        <div class="media-links container-h">
-            <button class="simple-button"><img src="images/twitter.png"></button>
-            <button class="simple-button"><img src="images/Linkedin icon.png" alt=""></button>
-            <button class="simple-button"><img src="images/medium.png" alt=""></button>
-            <button class="simple-button"><img src="images/github.png" alt=""></button>
-            <button class="simple-button"><img src="images/angellist.png" alt=""></button>
-        </div>
-    </main>
+        <ul class="media-links container-h">
+            <li><a href="#"><img src="images/twitter.png"></a></li>
+            <li><a href="#"><img src="images/Linkedin icon.png" alt=""></a></li>
+            <li><a href="#"><img src="images/medium.png" alt=""></a></li>
+            <li><a href="#"><img src="images/github.png" alt=""></a></li>
+            <li><a href="#"><img src="images/angellist.png" alt=""></a></li>
+        </ul>
```

Comment:
- [ Optional ] Kindly add your personal social media using `href` to let you connect your social networks with the icons.